ProcessLib/TwoPhaseFlowWithPrho/TwoPhaseFlowWithPrhoProcess.h

```diff
@@ -0,0 +1,79 @@
+/**
+ * \copyright
+ * Copyright (c) 2012-2017, OpenGeoSys Community (http://www.opengeosys.org)
+ *            Distributed under a Modified BSD License.
+ *              See accompanying file LICENSE.txt or
+ *              http://www.opengeosys.org/project/license
+ *
+ */
+
+#ifndef OGS_TWOPHASEFLOWWITHPRHOPROCESS_H
+#define OGS_TWOPHASEFLOWWITHPRHOPROCESS_H
+
+#include "MathLib/InterpolationAlgorithms/PiecewiseLinearInterpolation.h"
+#include "NumLib/DOF/LocalToGlobalIndexMap.h"
+#include "ProcessLib/Process.h"
+#include "TwoPhaseFlowWithPrhoLocalAssembler.h"
+
+namespace MeshLib
+{
+class Element;
+class Mesh;
+template <typename PROP_VAL_TYPE>
+class PropertyVector;
+}
+
+namespace ProcessLib
+{
+namespace TwoPhaseFlowWithPrho
+{
+/**
+ * \brief A class to simulate the two-phase flow process with P-P model in
```

Comment:
Please update all comments which were copy-pasted from other processes and are no longer valid.